.travis.yml

```diff
@@ -15,4 +15,4 @@ install:
 script:
   - vendor/bin/phpunit
   - bin/psalm
-  - vendor/bin/phpcs
+  - vendor/bin/php-cs-fixer fix --verbose --allow-risky=yes --dry-run
```

Comment:
it's intended to preserve the behaviour of  the [existing phpcs.xml](https://github.com/vimeo/psalm/blob/f65c618b2e2051edd25c9b0bd2431c19f20c92dc/phpcs.xml)